### PR TITLE
forms and login

### PR DESCRIPTION
**Pull request by LauDerek**
forms,
login,
signup

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -182,6 +182,8 @@ GEM
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 3.0)
       netrc (~> 0.7)
+    rollbar (1.4.5)
```
Comment (jkcheong92) [inline Gemfile.lock]: Did you add in a new gem for CSS?

File: app/views/merchants/registrations/new.html.erb [hunk 2]
```diff
@@ -1,30 +1,50 @@
-<div class="container">
-  <h2>Sign up </h2>
-  <br/><br/>
-  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-      <%= devise_error_messages! %>
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```
Comment (jkcheong92) [inline app/views/merchants/registrations/new.html.erb]: Why are you adding Bootstrap CDN file when it is already loaded in application.css?

File: app/views/merchants/registrations/new.html.erb [hunk 3]
```diff
@@ -1,30 +1,50 @@
...
 
-      <div class="field">
-        <%= f.label :email %><br />
-        <%= f.email_field :email%>
-      </div>
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
```
Comment (jkcheong92) [inline app/views/merchants/registrations/new.html.erb]: See above comment. Repeated script src will just slow down our system!

File: app/views/merchants/sessions/new.html.erb [hunk 4]
```diff
@@ -1,38 +1,61 @@
...
-      <% end %>
-  <% end %>
-
-  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-      <div class="field">
-        <%= f.label :email %><br />
-        <%= f.email_field :email, autofocus: true %>
-      </div>
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```
Comment (jkcheong92) [inline app/views/merchants/sessions/new.html.erb]: Dn need this again. Remove all bootstrap CDN!!!

File: app/views/form/_venue_create_form.erb [hunk 5]
```diff
@@ -1,149 +1,300 @@
...
 
       <div class = "row">
-        <div class="col-lg-4">
-          <p class="form-text clearfix">
-            <%= f.label "Street Address*" %>
-            <span>
-              <%= f.text_field :street %>
-            </span>
-          </p>
+        <div class="col">
```
Comment (jkcheong92) [inline app/views/form/_venue_create_form.erb]: Seems like you are replacing the default Rails form helper class with your CSS class - will this cause trouble later on? Are we restricted to using the new CSS classes for forms now?

**Comment by jkcheong92** [comment]
@LauDerek I only see venue_create_form. Where is deal_create form? And are the update_forms updated as well?

File: app/views/merchants/registrations/new.html.erb (hunk 2 above)
Comment (LauDerek) [inline app/views/merchants/registrations/new.html.erb]: Yeah ps this can be removed for the 3 updated pages

**Comment by LauDerek** [comment]
Deal create form not included yet for this pull request. Started with venue create will copy the same style over for deal form creation next.

**Comment by jkcheong92** [comment]
@LauDerek Just realized this: pls go update the forget password form css too.

**Pull request closed without merging**